test/e2e/tests/solana/web-socket-connection.spec.ts

```diff
@@ -0,0 +1,177 @@
+import { strict as assert } from 'assert';
+import { Suite } from 'mocha';
+import { Driver } from '../../webdriver/driver';
+import { withFixtures } from '../../helpers';
+import { ACCOUNT_TYPE } from '../../constants';
+import { loginWithBalanceValidation } from '../../page-objects/flows/login.flow';
+import HeaderNavbar from '../../page-objects/pages/header-navbar';
+import AccountListPage from '../../page-objects/pages/account-list-page';
+import FixtureBuilder from '../../fixture-builder';
+import LocalWebSocketServer from '../../websocket-server';
+
+describe('Solana Web Socket', function (this: Suite) {
+  it('a websocket connection is open when MetaMask full view is open', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder().build(),
+        title: this.test?.fullTitle(),
+        withSolanaWebSocket: true,
+        manifestFlags: {
+          remoteFeatureFlags: {
+            addSolanaAccount: true,
+          },
+        },
+      },
+      async ({ driver }: { driver: Driver }) => {
+        await loginWithBalanceValidation(driver);
+
+        const headerComponent = new HeaderNavbar(driver);
+        const accountListPage = new AccountListPage(driver);
+
+        await headerComponent.openAccountMenu();
+        await accountListPage.addAccount({
+          accountType: ACCOUNT_TYPE.Solana,
+          accountName: `Solana ${1}`,
+        });
```

Comment:
:information_source:  The Add Solana Account steps, can be skipped by using a fixture with an injected vault. This is left out of scope of this PR, but will be done in a subsequent task -- as it affects multiple tests